src/token/components/TokenRow.tsx

```diff
@@ -0,0 +1,69 @@
+import { memo, CSSProperties } from 'react';
+import { TokenRowReact } from '../types';
+import { formatAmount } from '../core/formatAmount';
+
+const styles = {
+  row: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    cursor: 'pointer',
+    padding: '4px 8px',
+  },
+  left: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  image: {
+    height: '32px',
+    width: '32px',
+    marginRight: '12px',
+  },
+  circle: {
+    height: '32px',
+    width: '32px',
+    borderRadius: '99999px',
+    background: 'gray',
+    marginRight: '12px',
+  },
+  column: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  label1: {
+    fontSize: '16px',
+    lineHeight: '1.5',
+    fontWeight: 500,
+    color: '#0A0B0D',
+  },
+  label2: {
+    fontSize: '16px',
+    lineHeight: '1.5',
+    fontWeight: 400,
+    color: '#5B616E',
+  },
+} as Record<string, CSSProperties>;
+
+export const TokenRow = memo(function TokenRow({ token, amount, onClick }: TokenRowReact) {
+  return (
+    <div data-testid="tokenRow" style={styles.row} onClick={() => onClick?.(token)}>
+      <div style={styles.left}>
+        {token.image === null ? (
+          <div data-testid="noTokenImage" style={styles.circle} />
```

Comment:
because we share `data-testid` with other components, let's as standard a prefix_ like `ockTokenRow_~~~`

so `ock` + `NameComponent` + `_` + `ElToTest`

a bit similar to what we did in https://github.com/coinbase/onchainkit/blob/main/src/identity/components/Avatar.tsx#L34

Reply:
what does `ock` stand for?